doc/Learn/Tutorials/Http/HowTo-Http.md

```diff
@@ -1,12 +1,101 @@
 ---
 uid: Learn.Tutorials.Http.HowToHttp
 ---
-# How-To: HTTP
+# How-To: Register an Endpoint for HTTP Requests
 
-This how-to is currently under construction. It will be available soon.
+When working with a complex application, centralized registration of your API endpoints is a good practice. This allows you to easily change the endpoint for a given service, and to easily add new services.
 
-## Work in progress 🚧
+## Step-by-steps
 
-### Have questions or feedback?
+### 1. Enable HTTP
 
-* Help us shape the documentation for this topic by providing feedback on the Uno.Extensions [repo](https://github.com/unoplatform/uno.extensions/discussions/categories/general)
\ No newline at end of file
+* Call the `UseHttp()` method to register a HTTP client with the `IHostBuilder` which implements `IHttpClient`:
+
+    ```csharp
+    private IHost Host { get; }
+
+    protected override void OnLaunched(LaunchActivatedEventArgs args)
+    {
+        var builder = this.CreateBuilder(args)
+            .Configure(host => {
+                host
+                .UseHttp( /* ... */);
+            });
```

Comment:
```suggestion
            .Configure(host => host
                .UseHttp( /* ... */)
            );
```
Most of the methods for configuring host builder are extension methods, so can be chained. This means you can avoid using braces here